sentry_sdk/scope.py

```diff
@@ -249,6 +252,26 @@ def clear_breadcrumbs(self):
         """Clears breadcrumb buffer."""
         self._breadcrumbs = deque()  # type: Deque[Breadcrumb]
 
+    def add_attachment(
```

Comment:
Similarly here, I'd be tempted to break this into two add_attachments_* methods to avoid the bytes/path ambiguity.

Reply:
The current proposed design was a single method. I do however assume we will need more than one in quite a few SDKs.

@HazAT @bruno-garcia @philipphofmann thoughts?

Reply:
Where overloads are a thing, we could have `addAttachment` with either: path, bytes or stream.
Not to end up in a place where more than 1 is passed in, or the caller gets confused.

Reply:
Going to resolve this then.

Reply:
resolving as duplicate of the open conversation.

Reply:
I'm sorry, I'm a bit lost here why we didn't add any overloads? Can you clarify this @mitsuhiko?